### DISCUSSION TITLE
Misc (not releases or shows) dates to be tracked

### DISCUSSION THREAD

**Discussion by WBTGS95** [kglw-dot-net/kglw-today#6] [General]
13 Dec 2008 - Tame Impala performs at Meredith Music Festival. Stu is in the crowd and is so inspired that he went home to start a psych band.

**Comment by WBTGS95** [comment]
24 Aug 2020 - Eric Moore announces his departure from the band to focus on Flightless Records full time, and the band return to self-publishing.

**Reply by alxndr** [reply]
wow I thought this was an earlier year than 2020

**Comment by WBTGS95** [comment]
18 Dec 2009 - Thee Oh Sees perform in Melbourne, the audience is mostly made up of future King Gizzard members who were all heavily inspired by the show.

**Reply by alxndr** [reply]
"mostly" ?? like there ten or fewer people in the audience, and more than half of them became The Gizz?

**Reply by WBTGS95** [reply]
Maybe 'largely' would be most accurate as there is no way of knowing. I was just paraphrasing Eric's description from the interview. He implied that they were all there, I would assume with other people who are not strictly within the current band but in the immediate circle that could have ended up in the band, and that there were less than 20 people total.

**Comment by WBTGS95** [comment]
25 Dec 2020 - The Official Bootlegger program is launched.

**Comment by WBTGS95** [comment]
24 Sep 2018 - The Flightless Records website infamously crashes due to extreme demand to pre-order vinyl reissues of the first five full length King Gizzard releases.

**Comment by WBTGS95** [comment]
19 Oct 2022 - KGLW.net goes live.

**Comment by WBTGS95** [comment]
3 Feb 2017 - Stu & Eric acknowledge the Gizzverse for the first time in a radio interview on Triple J. "[The albums] all exist in this parallel universe and they’re maybe from different times and different places but they all can co-exist in a meaningful way."

**Comment by WBTGS95** [comment]
26 Oct 2022 - On-demand streaming release of Sleeping Monster short film.

**Comment by WBTGS95** [comment]
23 Nov 2016 - King Gizzard win the Best Hard Rock/Heavy Metal Album ARIA Award for Nonagon Infinity. Other nominees included: Hellions, Parkway Drive, The Amity Affliction, and Twelve Foot Ninja.

**Comment by WBTGS95** [comment]
25 Nov 2020 - King Gizzard win the Best Hard Rock/Heavy Metal Album ARIA Award for Chunky Shrapnel. Other nominees included: Parkway Drive, Polaris, The Amity Affliction, and The Chats.

**Comment by WBTGS95** [comment]
9 Oct 2013 - King Gizzard win the first Carlton Dry Global Music Grant, allowing them to tour internationally for the first time.

**Comment by WBTGS95** [comment]
16 June 2022 - King Gizzard win the first Environmental Music Prize, and donate the winnings to The Wilderness Society.

**Comment by alxndr** [comment]
Nice, I whipped up something quick... https://github.com/kglw-dot-net/kglw-today/pull/8

<img width="1552" alt="Screenshot 2023-05-31 at 7 40 18 PM" src="https://github.com/kglw-dot-net/kglw-today/assets/174307/24a90d4f-53e1-444d-adc1-5b66371c4ab7">

**Reply by WBTGS95** [reply]
Sweet. I'm sure ideas for these will keep popping up, so once theres a place for them in the repo we can just do it with PRs

**Comment by alxndr** [comment]
I've added the dates here to the site, so closing this discussion now; we'll open a new discussion / issue with new dates.
